pages/plural-features/flows/scm-webhooks-and-pr-linking.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SCM Webhooks and PR Linking for Plural Flows
+description: Setting up SCM webhooks and automatically linking Pull Requests to Plural Flows
+---
+
+# Overview
```

Comment:
We should explain this more, in particular:

* it will vector index the changes in the PR for future retrieval to find a code-derived root cause of an alert that might be firing
* it will appropriately link to the owning Flow to ensure the investigation is bounded by the permission structure you've already established within Plural, ensuring our AI is appropriately governed according to your requirements.